src/CoreWCF.Primitives/src/CoreWCF/Channels/SecurityBindingElement.cs

```diff
@@ -0,0 +1,898 @@
+        {
+            return ((parameters is SecureConversationSecurityTokenParameters)
+                    || (parameters is SslSecurityTokenParameters)
+                    || (parameters is SspiSecurityTokenParameters));
+        }
+
+        internal virtual bool RequiresChannelDemuxer()
+        {
+            foreach (SecurityTokenParameters parameters in EndpointSupportingTokenParameters.Endorsing)
+            {
+                if (RequiresChannelDemuxer(parameters))
+                {
+                    return true;
+                }
+            }
+            foreach (SecurityTokenParameters parameters in EndpointSupportingTokenParameters.SignedEndorsing)
+            {
+                if (RequiresChannelDemuxer(parameters))
+                {
+                    return true;
+                }
+            }
+            foreach (SecurityTokenParameters parameters in OptionalEndpointSupportingTokenParameters.Endorsing)
+            {
+                if (RequiresChannelDemuxer(parameters))
+                {
+                    return true;
+                }
+            }
+            foreach (SecurityTokenParameters parameters in OptionalEndpointSupportingTokenParameters.SignedEndorsing)
+            {
+                if (RequiresChannelDemuxer(parameters))
+                {
+                    return true;
+                }
+            }
+            foreach (SupportingTokenParameters supportingParameters in OperationSupportingTokenParameters.Values)
+            {
+                foreach (SecurityTokenParameters parameters in supportingParameters.Endorsing)
+                {
+                    if (RequiresChannelDemuxer(parameters))
+                    {
+                        return true;
+                    }
+                }
+                foreach (SecurityTokenParameters parameters in supportingParameters.SignedEndorsing)
+                {
+                    if (RequiresChannelDemuxer(parameters))
+                    {
+                        return true;
+                    }
+                }
+            }
+            foreach (SupportingTokenParameters supportingParameters in OptionalOperationSupportingTokenParameters.Values)
+            {
+                foreach (SecurityTokenParameters parameters in supportingParameters.Endorsing)
+                {
+                    if (RequiresChannelDemuxer(parameters))
+                    {
+                        return true;
+                    }
+                }
+                foreach (SecurityTokenParameters parameters in supportingParameters.SignedEndorsing)
+                {
+                    if (RequiresChannelDemuxer(parameters))
+                    {
+                        return true;
+                    }
+                }
+            }
+            return false;
+        }
+
+        //void SetPrivacyNoticeUriIfRequired(SecurityProtocolFactory factory, Binding binding)
```

Comment:
Remove this commented code. If someone wants this feature, it can be added back in. It's purely an informational element in the WSDL and has zero functional impact on behavior.